assets/base/i18n/en.json

```diff
@@ -2495,6 +2495,7 @@
   "mobile.offlineIndicator.offline": "Cannot connect to the server",
   "mobile.open_dm.error": "We couldn't open a direct message with {displayName}. Please check your connection and try again.",
   "mobile.open_gm.error": "We couldn't open a group message with those users. Please check your connection and try again.",
+  "mobile.password_send.link": "If the account exists, a password reset email will be sent to:",
```

Comment:
As mentioned in my other comment, we could change the plain `password_send.link` to remove the HTML from it since it's something that can be translated and understood on its own so that we can also use it here, but that would need to be done in the web app repo first.